tests/lib/rules/lines-around-comment.js

```diff
@@ -1051,6 +1051,12 @@ ruleTester.run("lines-around-comment", rule, {
         {
             code: "foo\n/* this is pragmatic */",
             options: [{ applyDefaultIgnorePatterns: false, ignorePattern: "pragma" }]
+        },
+
+        // Hashbang comment
+        {
+            code: "#!comment\n\nvar a = 1;",
+            options: [{ afterHashbangComment: true }]
         }
```

Comment:
```suggestion
        },
        "#!comment\nvar a = 1;",
        {
            code: "#!comment\nvar a = 1;",
            options: [{}]
        },
        {
            code: "#!comment\nvar a = 1;",
            options: [{ afterHashbangComment: false }]
        },
        {
            code: "#!comment\nvar a = 1;",
            options: [{ afterLineComment: true, afterBlockComment: true }]
        }
```

A few tests to ensure that the rule doesn't require line after hashbang if `afterHashbangComment` isn't set to `true`.